src/sentry/api/endpoints/project_releases.py

```diff
@@ -16,6 +16,9 @@
 from sentry.utils.apidocs import scenario, attach_scenarios
 
 
+BAD_RELEASE_CHARS = '\n\f\t/'
```

Comment:
there is many other special chars like `\r` or `\0`, why not use proper regex allowing only the ones in ASCII range?